keps/sig-network/1860-kube-proxy-IP-node-binding/README.md

```diff
@@ -189,138 +189,76 @@ Yes. It is tested by `TestUpdateServiceLoadBalancerStatus` in pkg/registry/core/
+and running workloads consuming this service fails to reach it because of some 
+extra hop, or some misconfiguration on the LoadBalancer. 
 
-Be sure to consider highly-available clusters, where, for example,
-feature flags will be enabled on some API servers and not others during the
-rollout. Similarly, consider large clusters and how enablement/disablement
-will rollout across nodes.
--->
+A rollback can fail in case kube-proxy is not able to detect a rollback and re-add
+the LoadBalancer address back to the interface.
 
 ###### What specific metrics should inform a rollback?
 
-<!--
-What signals should users be paying attention to when the feature is young
-that might indicate a serious problem?
--->
+Workloads consuming a service configured to use the new `ipMode` and that starts
+to fail to reach the service, or an increase on the requests time are metrics that 
+should inform a rollback
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
-<!--
-Describe manual testing that was done and the outcomes.
-Longer term, we may want to require automated upgrade/rollback tests, but we
-are missing a bunch of machinery and tooling and can't do that now.
--->
+No.
 
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
 
-<!--
-Even if applying deprecation policies, they may still surprise some users.
--->
+No.
 
 ### Monitoring Requirements
 
-<!--
-This section must be completed when targeting beta to a release.
-
-For GA, this section is required: approvers should be able to confirm the
-previous answers based on experience in the field.
--->
-
 ###### How can an operator determine if the feature is in use by workloads?
 
-<!--
-Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
-checking if there are objects with field X set) may be a last resort. Avoid
-logs or events for this purpose.
--->
+As this is a low level operation, to check if it is working an operator should:
+* Verify a service of type=LoadBalancer and this feature enabled
+* Check and confirm that the IPs set on `.status.loadBalancer.ingress.ip` are not set
+on any interface
 
 ###### How can someone using this feature know that it is working for their instance?
 
-<!--
-For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
-for each individual pod.
-Pick one more of these and delete the rest.
-Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
-and operation of this feature.
-Recall that end users cannot usually observe component logs or access metrics.
--->
-
 - [ ] Events
   - Event Reason:
-- [ ] API .status
+- [X] API .status
   - Condition name:
-  - Other field:
+  - Other field: `.status.loadBalancer.ingress.ipMode` not null
```

Comment:
"is set to `Proxy`" not "is not null" right?

But that's not sufficient; this question _is_ about whether the feature is actually working, and checking `ipMode` only tells you whether the load balancer is _trying_ to use the feature, and doesn't tell you whether kube-proxy is doing the right thing with it. So you probably need to use `[ ] Other` here, and explain how the user can actually test the service IP to make sure it's using the new functionality.

Reply:
There may be two then: the API .status means some controller is using this field and setting it (so, working?) while your suggestion (which I agree!!) is "how can I know my traffic is going to the right place", right?

I will think on something here and add!